src/identity/mockUserData.ts

```diff
@@ -103,7 +104,6 @@ export const mockOrgDetailsArr: CurrentOrg[] = [
     clusterHost: 'https://fakehost.fakehost.fake',
     creationDate: '2022-06-08T16:59:44.827046Z',
     description: null,
-    isRegionBeta: false,
```

Comment:
This property isn't returned by this endpoint anymore, so it shouldn't be in the mock data.